app/components/ui/Footer.jsx

```diff
@@ -5,14 +5,40 @@ function Footer() {
   return (
     <footer className="site-footer" role="contentinfo">
       <div className="footer-content">
+        <div className="footer-top">
+          <section className="footer-service-guide">
+            SF SERVICE GUIDE
+          </section>
+          <section className="footer-links">
+            <ul>
+              <li>ABOUT US</li>
```

Comment:
Nit: My personal approach for this kind of stuff is to type out the text in the source code using normal casing and adding a CSS rule to set `text-transform: uppercase`. Also, I would probably make this first thing a heading tag like `<h1>/<h2>/...`, since it's more of a heading than a list item like the rest of the things.